addons/missionmodules/functions/fnc_playerBreath.sqf

```diff
@@ -0,0 +1,99 @@
+/*
+ * Author: joko // Jonas and Exikas
+ * Create a PFH for a Breathing Effect for player in closer distance
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * call ace_missionModules_fnc_playerBreath
+ *
+ * Public: Yes
+ */
+#include "script_component.hpp"
+
+if !(hasInterface) exitWith {};
+if !(GVAR(breathEnabled)) exitWith {};
+if !(local ACE_player) exitWith {};
```

Comment:
This could be combined into one statement.

Additionally if you make postInit a `clientInit` instead of `init` you don't need to check locality.